lucille-core/src/test/resources/ConfigValidationTest/badRunnerConfig.conf

```diff
@@ -0,0 +1,66 @@
+connectors: [
+  {
+    name: "connector1",
+    class: "com.kmwllc.lucille.connector.CSVConnector",
+    pipeline: "pipeline1",
+    idField: "id",
+    path: "classpath:RunnerTest/singleDoc.csv",
+  }
+  {
+    name: "connector2",
+    class: "com.kmwllc.lucille.connector.CSVConnector",
+    pipeline: "pipeline2",
+    idField: "id",
+    path: "classpath:RunnerTest/singleDoc.csv",
+  }
+]
+
+pipelines: [
```

Comment:
If this file is only to test a bad runner config, can we make the rest of the config simpler, like one connector with one pipeline that has one stage? Same comment for some of the other tests files that target one specific issue...